<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_certified</class>
 <widget class="QWidget" name="search_certified">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins Medium</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>650</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E5E5E5;
border-radius: 30px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: center;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>581</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #E5E5E5;
border-radius:0px;</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Word</string>
      </attribute>
      <widget class="QLineEdit" name="search_certifiedWordLineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>201</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #FFFFFF;
border-radius: 5px;
height: 35px
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Word:</string>
       </property>
      </widget>
      <widget class="QLabel" name="search_certifiedWordMeaningLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Meaning:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_certifiedUpdateButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>330</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_certifiedDeleteButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>330</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="search_certifiedWordMeaningBrowser">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>581</width>
         <height>183</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_certifiedWordSearchButton_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>42</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLabel" name="search_certifiedWordErrorLabel">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>80</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(197, 0, 0)</string>
       </property>
       <property name="text">
        <string>Enter a word</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Paragraph</string>
      </attribute>
      <widget class="QLabel" name="search_certifiedSentenceMeaningLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>271</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Words and their meanings:</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="search_certifiedSentenceMeaningBrowser">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>139</y>
         <width>581</width>
         <height>231</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_certifiedSentenceSearchButton">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLabel" name="search_certifiedSentenceErrorLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(197, 0, 0)</string>
       </property>
       <property name="text">
        <string>Enter a Sentence</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="search_certifiedSentenceLineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>6</y>
         <width>471</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #FFFFFF;
border-radius: 5px;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>501</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search a word or search a sentence for difficult words!</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1931</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E5E5E5;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>0</y>
       <width>337</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="search_certifiedNavSearch">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 5px;
background: #D1D1D1;
height: 30px;
width: 40px;
</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_certifiedNavInsert">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #E5E5E5;
border-radius: 5px;
</string>
        </property>
        <property name="text">
         <string>Insert a Word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_certifiedLogout">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #E5E5E5;
border-radius: 5px;
</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
